Added quality declaration (#90)
ros2/rmw_implementation: README.md

# rmw_implementation

> The decision which ROS middleware implementation should be used for C++.

Reviewer: I think this method does not take the decision, maybe a better wording would be using mapping/loadding
```suggestion
Implementation of ROS middleware functions used in C++.
```
